apps/openmw/mwgui/spellcreationdialog.cpp

```diff
@@ -147,7 +147,9 @@ namespace MWGui
 
         mDurationValue->setCaption("1");
         mMagnitudeMinValue->setCaption("1");
-        mMagnitudeMaxValue->setCaption("- 1");
+        static const std::string &to = MWBase::Environment::get().getWindowManager()->getGameSettingString("sTo", "-");
```

Comment:
This is technically correct since the lifetime of the string is extended to the lifetime of the reference. Still, looks odd and I can not see any benefit in using a reference here.